entrypoint.sh

```diff
@@ -90,18 +90,19 @@ exec cloud-api-adaptor-libvirt libvirt \
 }
 
 vsphere() {
-test_vars GOVC_USERNAME GOVC_PASSWORD GOVC_URL GOVC_DATACENTER GOVC_DATASTORE
+test_vars GOVC_USERNAME GOVC_PASSWORD GOVC_URL GOVC_DATACENTER
```

Comment:
I was about to suggest removing the GOVC_DATASTORE install/overlays/vsphere/kustomization.yaml but i see we missing it there, however, while here I'd suggest to add the GOVC_DATACENTER there as it's also missing and it is mandatory

Reply:
however, generally LGTM :)

Reply:
The next patch for issue #635 contains an updated kustomization.yaml as per my conversation with Pradipta https://github.com/confidential-containers/cloud-api-adaptor/pull/641#issuecomment-1448131203

 GOVC_DATACENTER is no longer mandatory if you request DRS. The next patch checks for that in NewProvider.